CHANGELOG.md

```diff
@@ -1,13 +1,57 @@
-## Parse Dashboard Changelog
-
-### master
-[Full Changelog](https://github.com/parse-community/parse-dashboard/compare/2.1.0...master)
-
-__New features:__
-* Added data export in CSV format for classes ([#1494](https://github.com/parse-community/parse-dashboard/pull/1494)), thanks to [Cory Imdieke](https://github.com/Vortec4800), [Manuel Trezza](https://github.com/mtrezza).
-* Collapse sidebar ([#1760](https://github.com/parse-community/parse-dashboard/pull/1760)), thanks to [Douglas Muraoka](https://github.com/douglasmuraoka), [Manuel Trezza](https://github.com/mtrezza).
-
-### 2.1.0
+# Parse Dashboard Changelog
+
+# master
+[Full Changelog](https://github.com/parse-community/parse-dashboard/compare/2.2.0...master)
+
+## New Features
+## Improvements
+## Fixes
+
+# 2.2.0
+[Full Changelog](https://github.com/parse-community/parse-dashboard/compare/2.1.0...2.2.0)
+
+## New Features
+- Added data export in CSV format for classes (Cory Imdieke, Manuel Trezza) [#1494](https://github.com/parse-community/parse-dashboard/pull/1494)
+- Added collapsing sidebar for small screens (Douglas Muraoka, Manuel Trezza) [#1760](https://github.com/parse-community/parse-dashboard/pull/1760)
+- Added exclude hidden columns from being fetched from server (itzharDev) [#1694](https://github.com/parse-community/parse-dashboard/pull/1694)
+- Added exclude hidden columns from being fetched from server (Prerna Mehra) [#1734](https://github.com/parse-community/parse-dashboard/pull/1734)
```

Comment:
I believe that, this pr was intended to solve a bug due to #1694 , which re-fetches the data only when a column is un-hidden  & its values are not cached.

Reply:
Well spotted, I've moved it down to fixes and changed the description.